esp/esp/program/modules/handlers/teacheracknowledgementmodule.py

```diff
@@ -5,16 +5,19 @@
 from django import forms
 from django.db.models.query import Q
 from esp.middleware.threadlocalrequest import get_current_request
+from esp.tagdict.models import Tag
 
 def teacheracknowledgementform_factory(prog):
     name = "TeacherAcknowledgementForm"
     bases = (forms.Form,)
     date_range = prog.date_range()
 
-    if date_range is None:
-        label = u"I have read the above, and commit to teaching my %s class." % (prog.program_type)
-    else:
-        label = u"I have read the above, and commit to teaching my %s class on %s." % (prog.program_type, date_range)
+    label = Tag.getTag('teacher_acknowledgement_text', target=prog)
```

Comment:
I don't think we even use this separate acknowledgement page anymore.  Now the acknowledgement is a required check box on the main teacher reg form (in a template override).  I say get rid of it.